PopPUNK/visualise.py

```diff
@@ -53,15 +53,18 @@ def get_options():
                              'to clusters [default = reference database directory]',
                         type = str)
     iGroup.add_argument('--previous-clustering',
-                        help='Directory containing previous cluster definitions '
+                        help='File containing previous cluster definitions '
```

Comment:
This choice makes sense! Can you update `docs/visualise.rst` to reflect this?

Reply:
Updated in 51b939f.